suite-native/app/metro.config.js

```diff
@@ -4,6 +4,8 @@ const nodejs = require('node-libs-browser');
 const { getSentryExpoConfig } = require('@sentry/react-native/metro');
 const { mergeConfig } = require('@react-native/metro-config');
 
+console.log(process.env.IS_DETOX_BUILD);
```

Comment:
If we really want the log here, there should be some explanation message prepended...

Reply:
removed